File: app/src/main/java/altcoin/br/decred/ReceiverOnBootComplete.java
package altcoin.br.decred;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import altcoin.br.decred.services.PriceAlertService;

public class ReceiverOnBootComplete extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        if (intent.getAction().equals(Intent.ACTION_BOOT_COMPLETED)) {
            context.startService(new Intent(context, PriceAlertService.class));
        }
    }
}

File: app/src/main/java/altcoin/br/decred/ReceiverOnBootComplete.kt
package altcoin.br.decred

import altcoin.br.decred.services.PriceAlertService
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class ReceiverOnBootComplete : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == Intent.ACTION_BOOT_COMPLETED) {
            context.startService(Intent(context, PriceAlertService::class.java))
        }
    }
}
